zhaquirks/tuya/tuya_motion.py

```diff
@@ -85,6 +85,14 @@ class TuyaPresenceStateV02(t.enum8):
     Gigantic = 0x05
 
 
+class TuyaPresenceStateV03(t.enum8):
+    """Tuya presence state enum, variation 03."""
+
+    Unoccupied = 0x00
+    Presence = 0x01
+    Move = 0x02
```

Comment:
Can we call this `Movement`?

```suggestion
    Movement = 0x02
```